proxies/live/apiproxy/resources/jsc/SetnRemoveVersion.js

```diff
@@ -16,5 +16,3 @@ for (var i = 0; i < acceptArray.length; i++) {
 }
 
 context.setVariable('versionNumber', versionNumber);
-
-context.setVariable('request.header.Accept', modifiedAcceptHeader);
```

Comment:
The `modifiedAccpetHeader` variable is unused. Can you please remove the code and that `else` branch as well. They are all dead code now.